src/templates/playgroundTemplates.ts

```diff
@@ -0,0 +1,17 @@
+const defaultTemplate: string = `//select the database to use
```

Comment:
Just for readability for a template I think making the comments complete sentences might read smoother. 
`// Select the database to use.`
`// Run a find command.`